tensorboard/webapp/runs/views/runs_table/runs_data_table.ts

```diff
@@ -127,4 +127,10 @@ export class RunsDataTable {
     const input = event.target! as HTMLInputElement;
     this.onRegexFilterChange.emit(input.value);
   }
+
+  trackByRuns(index: number, data: TableData) {
```

Comment:
Can you document what/why you're doing this? And can you add a test?

Reply:
I'll add a comment explaining how `trackBy` guards rerenders of items within an `ngFor`.
This is going to be very hard to test because it involves tracking whether a non angular component is rerendered.
I can add a test ensuring this get called, but since it's a framework feature I'm not sure that is worth doing.

Reply:
Maybe just a test for the logic in this function. it('returns the data object without the color attribute');